ios/Plugin/Plugin.swift

```diff
@@ -6,19 +6,24 @@ import Sentry
 public class SentryCapacitor: CAPPlugin {
     
     @objc public func startWithOptions(capOptions: CAPPluginCall) {
-        SentrySDK.start { options in
-            options.dsn = capOptions.getString("dsn") ?? ""
-            options.debug = capOptions.getBool("debug") ?? false
-            options.environment = capOptions.getString("environment") ?? "production"
-            options.enableAutoSessionTracking = capOptions.getBool("enableAutoSessionTracking") ?? false
-            options.sessionTrackingIntervalMillis = capOptions.options["sessionTrackingIntervalMillis"] as? UInt ?? 5000
-            options.attachStacktrace = capOptions.getBool("attachStacktrace") ?? false
+        let options = capOptions;
+        
+        options.beforeSend = { event in
+            if (event.exceptions.firstObject.type != nil && ["Unhandled JS Exception"].location != NSNotFound) {
```

Comment:
This assumes it has the same type as it does on React Native. Might not be the same on Capacitor.

Also, does it assume there's always one exception and it has a type field?

`event.exceptions.firstObject.type ` maybe we want to check for nulls here (might be also relevant on RN /cc @jennmueng )

Reply:
Hmm this is straight from React Native, I've never seen any issues with it though. But yeah we should do a null check just in case.

Reply:
You mean no one complained about it, right? :D
Because if it fails there it means the event would be lost and the developer wouldn't know about it.

Reply:
So over in RN the iOS native module is in objective-c and I just found out it has nil messaging, so accessing `event.exceptions.firstObject.type` shouldn't throw anything.

However here on swift you might want to use optional chaining `?.` to get the same behavior.